releasenotes/notes/remove-qiskit-algorithms-a43541fe24b72208.yaml

```diff
@@ -0,0 +1,9 @@
+---
+upgrade:
+  - |
+    The ``qiskit.algorithms`` module has been removed, following its deprecation in
+    Qiskit 0.44. The code has been migrated to a standalone library (``qiskit_algorithms``)
+    and can be found on PyPi or `GitHub <https://github.com/qiskit-community/qiskit-algorithms>`_.
+    The decision to migrate the algorithms module to a separate package
+    was made to clarify the purpose Qiskit and make a distinction between the tools
+    and libraries built on top of it.
```

Comment:
Not all qiskit.algorithms was migrated. Only the algorithm code that uses primitives. I think it was mentioned in prior release notes that if any of opflow/Quantum instance based algorithm function was still being used a user would first need to migrate to the primitive based algorithms. Should we briefly mention this aspect again here? Or perhaps minimally change `The code has been migrated...` to `The primitive based algorithms from this code have been migrated...` or something like that? though perhaps that's less clear if you had not been paying attention and things still worked!